### PR TITLE
Hydra

### PR DESCRIPTION
**Pull request by mizaqq**
PR contains moving config to hydra package:
- refactored main.py to get config from hydra instead of using args or jsons
- moved configs to hydra tree to add a possibility to run experiments with different configs
- added multirun to execute code on all 3 types of models

### CODE REVIEW

File: model/main.py [hunk 1]
```diff
@@ -1,64 +1,52 @@
...
+def main(cfg: DictConfig) -> None:
+    print(OmegaConf.to_yaml(cfg))
     uri = os.environ.get("MLFLOW_TRACKING_URI", "http://48.209.80.111:5000")
     experiment_name = os.environ.get("MLFLOW_EXPERIMENT_NAME", "housepricesml")
     mlflow_handler = MLFlowHandler(uri, experiment_name)
-    df = pd.read_csv(args.train_data)
+    df = pd.read_csv(cfg["data"]["train"])
     uncorrelated_col, insignificant_col, missing_values_col = get_data_for_preprocessing(
-        df, treshhold=0.05, mlflow=mlflow_handler
+        df, treshhold=cfg["preprocessing"]["threshhold"], mlflow=mlflow_handler
```
Comment (adamwawrzynski) [inline model/main.py]:
```suggestion
        df, treshhold=cfg["preprocessing"]["threshold"], mlflow=mlflow_handler
```

File: model/conf/config.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+data:
+  train: 'model/data/train.csv'
+
+preprocessing:
+  threshhold: 0.05
```
Comment (adamwawrzynski) [inline model/conf/config.yaml]:
```suggestion
  threshold: 0.05
```

**Review by adamwawrzynski — APPROVED** [review]
LGTM

File: model/main.py (hunk 1 above)
Comment (mizaqq) [reply model/main.py]: fixed [0408cb9](https://github.com/mizaqq/housepricesml/pull/4/commits/0408cb9a3917dec9b316787c48d676a45d2a3d4c)

File: model/conf/config.yaml (hunk 2 above)
Comment (mizaqq) [reply model/conf/config.yaml]: fixed [0408cb9](https://github.com/mizaqq/housepricesml/pull/4/commits/0408cb9a3917dec9b316787c48d676a45d2a3d4c)

**Pull request merged**